docs/developer-guide/quick-start-guide.md

```diff
@@ -0,0 +1,116 @@
+
+This tutorial is the quickest way to get started with Origami. By the end of it you should know enough to create disposable prototypes using Origami's ui modules. If you want to know how to use Origami in production websites, you'll need a bit more information than this tutorial includes.
+
+We have a more detailed set of tutorials if you want a proper introduction. [Take me to the proper guide](/docs/developer-guide/modules/).
+
+## 1. Set up your page
+
+We're going to use [Codepen.io](http://codepen.io/pen/?editors=1000) for this demo. You could also use [JS Bin](http://jsbin.com/?html,output) or serve the demo from a local server.
+
+First, let's put some minimum viable HTML onto the page. This is not the HTML you would use in production, but for prototyping with the Build Service it's fine.
+
+
+	<html>
+		<head>
+			<!-- build service links and scripts here -->
+		</head>
+		<body>
+			Hello!
+			<button>Here's a button</button>
+		</body>
+	</html>
+
+
+
+## 2. Include the CSS
+
+OK, we've got a button there, but it's completely unstyled. Let's use the `o-buttons` Origami module to style it.
+
+We're going to use the Build Service to provide the styles.
+
+To do that we need to request the `o-buttons` CSS from the Build Service:
+
+```
+https://build.origami.ft.com/v2/bundles/css?modules=o-buttons
+```
+
+This request says "give me the CSS for o-buttons at the latest version"
+
+Let's add that to our page in the `<head>` as a `<link ...>` tag
+
+```
+<link rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-buttons">
+```
+
+<aside class='read-more'>
+<strong>Read more about component versioning</strong>
+<p>This example only uses the component tag (o-buttons). To use Origami modules in production code safely, you should also use a version number with your Build Service request.</p>
+<a class='o-buttons' href='{{site.baseurl}}/docs/developer-guide/modules/module-versioning/'>Find out more about component versioning</a>
+</aside>
+
+## 3. Add the classes
+
+Next add the right classes to your HTML. Some Origami components are more complicated than `o-buttons`. To help with this we provide example HTML already marked up with the right classes that you can copy into your project.
+
+For this tutorial though, you need to add the class `o-buttons` to the `<button>` tag.
+
+o-buttons has some design variations. Let's apply the `standout` variation by the adding `o-buttons--standout` class.
+
+```
+<button class="o-buttons o-buttons--standout">Here's a button</button>
+```
+
+## 4. Include the JavaScript
+
+`o-buttons` also has some JavaScript to make buttons work better on touch devices like phones. Getting JavaScript from the build service is as easy as getting the CSS.
+
+```
+https://build.origami.ft.com/v2/bundles/js?modules=o-buttons
+```
+
+Instead of using a `<link ...>` tag, use a `<src ...>` tag.
+
+```
+<src rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-buttons">
```

Comment:
Should this be: `<script type="javascript" src="https://build.origami.ft.com/v2/bundles/js?modules=o-buttons">` ?